src/js/veteran-id-card/containers/VeteranIDCard.jsx

```diff
@@ -72,9 +73,7 @@ const mapStateToProps = (state) => {
     profile: userState.profile,
     loginUrl: userState.login.loginUrl,
     verifyUrl: userState.login.verifyUrl,
-    rateLimitAuthed,
-    rateLimitUnauthed,
-    serviceRateLimitedRandomizer
+    vicSettings
```

Comment:
I pass these to props only to expose them to React dev tools since we aren't planning on moving these into Redux.  If we wanted to bind these to the class instead that would make sense to me too.